File: java/pruebas/src/main/java/com/test/Calculadora/Calculadora.java
package com.test.Calculadora;

import java.util.Scanner;

public class Calculadora {
    public Scanner scanner;
    public int suma,resta,multi,divi;
    private int num;
    private int can;

    public Calculadora(){
        scanner = new Scanner(System.in);
        System.out.println("Ingresa una cantidad de numeros para sumar:");
        can = scanner.nextInt();
    }

    public void suma(){
        suma = 0;
        System.out.println();
        for (int i = 0; i < can; i++) {
            System.out.println("Ingresa un numero:");
            num = scanner.nextInt();
            suma += num;
        }
        System.out.println();
        System.out.println("La suma total es: " + suma);
    }


    public void resta(){
        resta = 0;
        System.out.println();
        for (int i = 0; i < can; i++) {
            System.out.println("Ingresa un numero:");
            num = scanner.nextInt();
            resta -= num;
        }
        System.out.println();
        System.out.println("La resta total es: " + resta);
    }
}

File: python/practica/ejercicios_p/calculadora.py
class calculadora:
    def __init__(self, num='',can=0,guardar=None):
        self.num = num
        self.can = can
        self.guardar = [] if guardar is None else guardar
        
        
    def ingresar(self):
        self.can = int(input('Ingrese la cantidad de numeros que desea operar: '))
        for i in range(self.can):
            self.num = int(input('Ingrese un numero:'))
            self.guardar.append(self.num)
    
    def mostrar(self):
        print(f'Los numeros que se va a operar son {self.guardar}')
        
        
class suma(calculadora):
    def sumar(self):
        resula


def main():
    opr = calculadora()
    opr.ingresar()


if __name__ == "__main__":
    main()